ballerina/listener.bal

```diff
@@ -34,37 +34,37 @@ public isolated class Listener {
     # + s - The service to be attached.
     # + name - The name of the service.
     # + return - An error if the attachment fails.
-    public isolated function attach(Service s, string[]|string? name = ()) returns error? =
+    public isolated function attach(Service s, string[]|string? name = ()) returns Error? =
```

Comment:
This is an interop call have we checked whether we return a proper error from the native code as well ?

Reply:
As per the offline discussion we had we have already done the necessary changes in the native-level. Hence, +1.